helix-term/src/commands/typed.rs

```diff
@@ -895,6 +895,23 @@ fn yank_main_selection_to_clipboard(
     yank_main_selection_to_clipboard_impl(cx.editor, ClipboardType::Clipboard)
 }
 
+fn yank_joined(
+    cx: &mut compositor::Context,
+    args: &[Cow<str>],
+    event: PromptEvent,
+) -> anyhow::Result<()> {
+    if event != PromptEvent::Validate {
+        return Ok(());
+    }
+
```

Comment:
I don't think we do this consistently in typable commands but we can give some feedback when the command gets more arguments than expected:

```rust
ensure!(args.len() <= 1, ":yank-join takes at most 1 argument");
```

Reply:
Just updated it! Would it be helpful to make a pr to add this to the other commands as necessary?

Reply:
Yeah I think it would be good to have some feedback especially on the new/save file operations like `new`, `write`, `format`, etc 👍. Discarding arguments silently seems potentially confusing to me